python/lsst/drp/tasks/assemble_coadd.py

```diff
@@ -495,8 +511,13 @@ def prepareInputs(self, refList):
         tempExpRefList = []
         weightList = []
         imageScalerList = []
+        outputPsfMatchedWarpRefList = []
+
+        if not psfMatchedWarpRefList:
+            psfMatchedWarpRefList = (None for _ in range(len(refList)))
```

Comment:
I think it's better to default to list comprehension rather than generator expression; lists just behave less surprisingly and are usually more performant when the thing you're iterating over isn't absolutely enormous.

Reply:
Hmm, I thought I had it changed to list comprehension.

Reply:
Ah, I did review some of this commit-by-commit, so maybe I just commented on code you already changed before you were done.